### PR TITLE
Added support for OV 5.2 for cluster profile resources

### PR DESCRIPTION
**Pull request by VenkateshRavula**
### Description
Added support for OV 5.2 for cluster profile resources

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# 5.2.0 (unreleased)
+Extends support of the SDK to OneView REST API version 1600 (OneView v5.20).
```
Comment (sijeesh) [inline CHANGELOG.md:2]: Add the list of resources supported in this version

File: hpOneView/resources/hypervisors/hypervisor_cluster_profiles.py [hunk 2]
```diff
@@ -102,3 +103,26 @@ def get_compliance_preview(self):
         """
         compliance_uri = "{0}/compliance-preview".format(self.data["uri"])
         return self._helper.do_get(compliance_uri)
+
+    def delete(self, timeout=-1, softDelete=False, force=False):
```
Comment (sijeesh) [inline hpOneView/resources/hypervisors/hypervisor_cluster_profiles.py:107]: variables name style should be consistent.

File: CHANGELOG.md (hunk 1 above)
Comment (VenkateshRavula) [reply CHANGELOG.md:3]: done

**Comment by VenkateshRavula** [comment]
> It's a good practice to open separate PRs for resources even it's a minor change. This will make the review process easy and error-free.

Please review for now. Will raise seperate PRs from next time.

File: hpOneView/resources/hypervisors/hypervisor_cluster_profiles.py (hunk 2 above)
Comment (VenkateshRavula) [reply hpOneView/resources/hypervisors/hypervisor_cluster_profiles.py]: changed softDelete to soft_delete

**Pull request merged**